Project/Project/wwwroot/css/site.css

```diff
@@ -69,3 +69,17 @@ body {
   white-space: nowrap;
   line-height: 60px; /* Vertically center the text there */
 }
+
+
+table {
+    border: 1px solid GREEN;
+    padding: 5px 10px;
+}
+
+h1 {
```

Comment:
Changed header fond size and background color

Reply:
Does this make contrast better or worse?  That's an important factor to document here.  If worse, it's adding to, and not reducing, technical debt, as it is an accessibility issue.